app/views/catalog/_metadata_button_default.html.erb

```diff
@@ -0,0 +1,14 @@
+<% if feature_flags.view_all_metadata? %>
```

Comment:
Now that we have the MODS display stuff working, do we need the feature flag still?

Reply:
Great idea. I've pulled that out now.